samples/graph-notification-team-channel/csharp/README.md

```diff
@@ -0,0 +1,104 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates use of team/Channel subscription that will post notifications when user create/edit/delete.
```

Comment:
This is a sample application which demonstrates the use of team/Channel subscription that will post notifications when user create/edit/delete team/channel.

Reply:
fixed